### PR TITLE
Fix spring cloud

### PR DESCRIPTION
**Pull request by naah69**
// Describe your PR here; eg. Fixes #issueNo

issue #1192 

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-spring-boot-starter/soul-spring-boot-starter-sync-data-center/soul-spring-boot-starter-sync-data-nacos/pom.xml [hunk 1]
```diff
@@ -42,5 +42,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba.nacos</groupId>
+            <artifactId>nacos-client</artifactId>
+            <scope>provided</scope>
+        </dependency>
```
Comment (dengliming) [inline soul-spring-boot-starter/soul-spring-boot-starter-sync-data-center/soul-spring-boot-starter-sync-data-nacos/pom.xml:49]: Please check the dependencies in `pom.xml`. The `soul-spring-boot-starter-sync-data-nacos` depends `soul-sync-data-nacos` and the `soul-sync-data-nacos `depends `nacos-client`

**Review by dengliming — CHANGES_REQUESTED** [review]

**Comment by yu199195** [comment]
no we can not merged

File: soul-spring-boot-starter/soul-spring-boot-starter-sync-data-center/soul-spring-boot-starter-sync-data-nacos/pom.xml (hunk 1 above)
Comment (naah69) [reply soul-spring-boot-starter/soul-spring-boot-starter-sync-data-center/soul-spring-boot-starter-sync-data-nacos/pom.xml:49]: yeah,the `provided` just make build pass

**Comment by naah69** [comment]
> no we can not merged



> no we can not merged

why

**Comment by naah69** [comment]
> no we can not merged

the core of pr is that fix springcloud error

**Comment by yu199195** [comment]
serviceId is field not is selector name

**Comment by naah69** [comment]
> serviceId is field not is selector name

so that's it，but i can't find the field in handle of selectorData ,then it error.
where is the field

**Comment by yu199195** [comment]
@naah69  is this

SpringCloudSelectorHandle selectorHandle = GsonUtils.getInstance().fromJson(selector.getHandle(), SpringCloudSelectorHandle.class)

**Comment by yu199195** [comment]
selector.getHandle()  is the application name

**Comment by naah69** [comment]
> @naah69  is this
> 
> SpringCloudSelectorHandle selectorHandle = GsonUtils.getInstance().fromJson(selector.getHandle(), SpringCloudSelectorHandle.class)

but i can't  find the field in handle.so it error

**Comment by naah69** [comment]
soul.client.props.appName didn't set

**Comment by yu199195** [comment]
String appName = env.getProperty("spring.application.name");   please update master

**Pull request closed without merging**